### ISSUE TITLE
duplicate PDSA cycles show up in the table (practice page)

### ISSUE THREAD

**Issue by squabwrangler** [seemack/healthy-hearts-northwest-dev#87]
The PDSA section for St. Luke’s Clinic: EOMA has three PDSAs that are repeated and I am not allowed to delete as much as I try.  -Angie Combe

**Comment by rpardee** [comment]
Confirmed.  [This is the page](http://healthy-hearts-northwest.herokuapp.com/coach_practices/17770).

Currently shows:

![image](https://cloud.githubusercontent.com/assets/26327/14969424/36cdda5c-1077-11e6-8e89-c35c5ff836bc.png)

**Comment by seemack** [comment]
A HIT-only contact got the PDSAs associated forwarded to it, though they're never displayed on the HIT-specific page.  Have to think about which of these is the best approach:

(1) Don't show historical PDSAs. This is simplest, but if coaches like to see them, I don't want to remove useful functionality.
(2) Filter PDSAs. Basically, unduplicate on the fly.  This will take processing time and it's questionable how we'll identify duplicates because the new records will have new IDs.
(3) Don't allow HIT-only visits to have PDSAs.  The most logical solution, but it complicates the process of copying PDSAs forward because now the last visit (if it's HIT only) may not have any PDSAs.

(3) is probably the best solution, but it involves the most programming.  If anyone is OK with (1) or (2), let me know.

**Comment by seemack** [comment]
Hmm, issue #91 is the mirror image of this.  A HIT-only visit was entered out of order - a January visit entered in April - with no PDSAs, so no PDSAs were brought forward.  It looks like HIT visits aren't integrated well with the PDSA functionality at all.  I'll have to rethink the whole thing.  :(

**Comment by vanborkulo** [comment]
HIT only visits - there isn't an option to input PDSAs.  those visits only have 'topics discussed' and 'comments'

**Comment by seemack** [comment]
Right, but the data is moving forward anyway.  We were designing the PDSA carry-forward and the HIT-only visits in parallel, and never discussed how they'd interact.  Let's clarify on Wednesday.

I think this can still be closed as a duplicate of #91?
